tests/testthat/test-geom-violin.R

```diff
@@ -46,6 +46,14 @@ test_that("quantiles do not fail on zero-range data", {
   expect_equal(length(layer_grob(p)), 1)
 })
 
+test_that("quantiles do not issue warning", {
+  df <- data.frame(x = 1, y = c(0, 0.25, 0.5, 0.75, 5))
+
+  p <- ggplot(df, aes(x = x, y = y)) +
+    geom_violin(draw_quantiles = c(0.5))
+
+  expect_warning(plot(p), regexp = NA)
```

Comment:
Can you please either use `expect_snapshot_warning()` here, or include a `regexp` to confirm that you're matching the expected warning?

Reply:
`expect_warning(..., regexp = NA)` asserts that there should be no warnings, so the lack of a regexp is intentional (the point of this PR is to remove a warning), compare https://stackoverflow.com/a/33638939. Do you see a way of achieving that with `expect_snapshot_warning()`?

Reply:
Oh sorry, I missed that.